solutions/src/main/java/ru/sberbank/school/task02/Client.java

```diff
@@ -0,0 +1,110 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.exception.ConverterConfigurationException;
+import ru.sberbank.school.task02.exception.WrongSymbolException;
+import ru.sberbank.school.task02.util.*;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+
+public class Client implements FxClientController {
+    private ExtendedFxConversionService converter;
+    private String beneficiary;
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm");
+
+    public Client(ExternalQuotesService externalQuotesService) {
+        this.beneficiary = System.getenv("SBRF_BENEFICIARY");
+        ServiceFactoryImpl factory = new ServiceFactoryImpl();
+        this.converter = factory.getExtendedFxConversionService(externalQuotesService);
+    }
+
+    public static void main(String[] args) {
+        if (args.length < 3) {
+            throw new ConverterConfigurationException("Not enough input parameters");
+        }
+
+        ExternalQuotesService externalQuotesService = new ExternalQuotesServiceDemo();
+
+        Client client = new Client(externalQuotesService);
+
+        Map<String, String> params = new HashMap<>();
+
+        for (String arg : args){
```

Comment:
WhitespaceAround: &#39;{&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-207021740>*